docs/website/docs/getting-started.md

```diff
@@ -10,35 +10,47 @@ Please make sure you have [installed `dlt`](./installation.mdx) before getting s
 
 ## 1. Initialize project
 
-Create a `dlt` project with a pipeline that loads data from the chess.com API to Google BigQuery by running:
+Create a `dlt` project with a pipeline that loads data from the chess.com API to DuckDB by running:
 
 ```
-dlt init chess bigquery
+dlt init chess duckdb
 ```
 
-Install the dependencies necessary for Google BigQuery:
+Install the dependencies necessary for DuckDB:
 ```
 pip install -r requirements.txt
 ```
 
-## 2. Set up Google BigQuery
+## 2. Run pipeline
 
-Follow [BigQuery setup](./destinations#google-bigquery) to create the service account credentials you'll need to add to `.dlt/secrets.toml`.
+Run the pipeline to load data from the chess.com API to DuckDB by running:
+```
+python3 chess.py
+```
 
-## 3. Run pipeline
+## 3. Query DuckDB database
 
-Run the pipeline to load data from the chess.com API to Google BigQuery by running:
+Create a `query.py` file where you can write SQL queries:
 ```
-python3 chess.py
+import duckdb
+
+conn = duckdb.connect('quack.duckdb')
```

Comment:
Yeah we'll move the `quack` to the working folder - so you can do the above. There's however a way that will work with any destination and does a lot of things behind the scenes for you. here's the script. take what you want from there
```python
import dlt

# this will use the same configuration as in `chess.py` and open the connection seamlessly. it is also destination independent way
# also:
# - it will let you query tables without schema prefix 
# - will apply the same settings to duckdb you had when loading
# - will inject s3 and other credentials so you can work with s3 bucket automatically
# - etc.

with dlt.attach(pipeline_name="chess").sql_client() as client:
    # this works on any destination
    rows = client.execute_sql('SELECT COUNT(*) FROM players_games')
    print(rows)
    
    # this works on any destination
    with client.execute_query('SELECT * FROM players_games LIMIT 10') as query:
        
        # print(query.fetchall())
        # this is duckdb specific
        print(query.fetch_df())

    # or call the duckdb directly
    client.native_connection.sql('SELECT COUNT(*) FROM players_games').show()
    # next dlt version will allow for this (no native_connection necessary)
    # client.sql('SELECT COUNT(*) FROM players_games').show()
```

Reply:
I am quite skeptical that we should be recommending folks to interact with `DuckDB` through `dlt`. The things you mentioned are nice, but I'm not sure they are worth the complexity / confusion they would introduce to our users (at least for `DuckDB`)

Reply:
I'm totally for it if we move this into a place dedicated to `duckdb`. if this is getting started section, we should IMO show stuff that works with any destination.
Do we expect mostly duckdb crowd here? if this is so then keep your original script, just fix the table name you select from. we may put something more universal later

I'll be pushing the db location fix on monday

Reply:
That would make sense to me if the way the user should interact with their destinations is through `dlt`. I don't think this should be the recommended way. I think `dlt` should be a simple, easy data loading library and not try to do too much. We can have this functionality since there are use cases where you need to query in order to load, but I don't think we should train folks to use `dlt` interfaces for their daily querying, their transformations, etc.

I would remove the script and just use the `dlt show` part for now

Reply:
This is old UNIX principle of doing just one thing and doing this well :) `sql_client` is used mostly internally or to quickly test/fetch data in the same script that ran pipeline. people should use something else for transformations.

just note that `duckdb` is special because you see your database file and it is easy to connect. the user of `duckdb` also comes here for a database they love. but think about bigquery user which has no clue about the database and querying data in python. to actually use the data:
- you need to find which of hundreds objects is actual google sql client
- learn a special syntax that is different from python dbapi syntax (or dig out that there's dbapiconnection in google namespace)
- learn how to authenticate with service json, load the file and pass it somewhere
- that to query your dataset you need to use project_id.dataset_name.table_name enclosed in non standard quotation
- that there's a special `to_dataframe` helper that will execute query and convert it to a dataframe (if you install special deps)

this is all eliminated by dlt that is reusing the credentials you are supplied by loading and will return you an sql_client which is just bigquery client with a few standard methods attached

long rant heh but there are situations that sql_client makes a lot of sense